City-of-Helsinki/infraohjelmointi-api
# HKISD-134: add indexes for phases

- added a field for projectPhase table for defining the order of phases
### How to test:
- run migration by running `python manage.py migrate` inside the docker container to add the new field to the table
- go inside the database by running `psql $DATABASE_URL`
- add indexes with the sql file by running `\i update-phase-indexes.sql`
- you can see that the indexes are in the table and that they are correct by running `select * from infraohjelmointi_api_projectphase;`
- you can also add a console.log to the frontend's listsService's getProjectPhases function to see that the indexes are coming to the frontend as well
